lesson4/homeworklesson4.py

```diff
@@ -0,0 +1,76 @@
+def creating_a_string(first_string:str, second_string:str) -> str:
+    result_string = []
+    second_string = second_string[::-1]
+    if len(first_string) - len(second_string) > 0:
+        n = first_string[-(len(first_string) - len(second_string))::1]
+        for first_string, second_string in zip(first_string, second_string):
+            result_string.append(first_string + second_string)
+        result_string.append(n)
+    elif len(first_string) - len(second_string) < 0:
+        n = second_string[(len(first_string) - len(second_string))::1]
+        for first_string, second_string in zip(first_string, second_string):
+            result_string.append(first_string + second_string)
+        result_string.append(n)
+    else:
+        for first_string, second_string in zip(first_string, second_string):
+            result_string.append(first_string + second_string)
+    result_string = "".join(result_string)
+    return result_string
+
+def sort_list_of_int(test_list:list[int], condition:bool) -> list[int]:
+    for n in range(len(test_list)):
+        for a in range(len(test_list) - n - 1):
+            if test_list[a] > test_list[a + 1]:
+                test_list[a], test_list[a + 1] = test_list[a + 1], test_list[a]
+    if condition == True:
```

Comment:
Вот так делать очень плохо. У тебя итак булевая переменная, зачем ее сравнивать с правдой? 

Тут достаточно просто написать 
```suggestion
    if condition:
```